lib/karafka/setup/config.rb

```diff
@@ -37,7 +37,7 @@ class Config
         # Connection pool size for producers. Note that we take a bigger number because there
         # are cases when we might have more sidekiq threads than Karafka routes (small app)
         # or the opposite for bigger systems
-        setting :size, -> { [::Karafka::App.routes.count, Sidekiq.options[:concurrency]].max }
+        setting :size, [::Karafka::App.routes.count, Sidekiq.options[:concurrency]].max
```

Comment:
We can, I think. It's default value for size setting. Added validator about size which should be an integer.